src/metric.cc

```diff
@@ -130,8 +134,12 @@ Metric::SendIncrementRequest(const double& value)
     operation_value_ = value;
     SaveToSharedMemory(stub->ShmPool());
     CustomMetricsMessage* custom_metrics_msg = nullptr;
-    stub->SendCustomMetricsMessage(
-        &custom_metrics_msg, PYTHONSTUB_MetricRequestIncrement, shm_handle_);
+    AllocatedSharedMemory<CustomMetricsMessage> custom_metrics_shm;
+    std::unique_ptr<IPCMessage> ipc_message;
+
+    stub->SendMessage<CustomMetricsMessage>(
+        ipc_message, &custom_metrics_msg, custom_metrics_shm,
```

Comment:
I think we no longer need to have custom_metrics_msg separately. It can be created from `custom_metrics_shm`.